### ISSUE TITLE
Add `lon` and `lat` columns to USGS and COOPS stations dataframes

### ISSUE THREAD

**Issue by pmav99** [oceanmodeling/searvey#114]
We do have this information via the `geometry` column. Nevertheless, it might be convenient to have them also as separate columns. IOC already does have both.

@SorooshMani-NOAA thoughts?

**Comment by SorooshMani-NOAA** [comment]
Makes sense, I don't have any strong thoughts against it. I'll put this on my todo list. Do we have any urgency for this?

**Comment by pmav99** [comment]
No not really. We just found ourselves adding those columns when comparing some data
